optaweb-employee-rostering-frontend/src/store/tenant/operations.ts

```diff
@@ -80,6 +87,11 @@ ThunkCommandFactory<void, RefreshTenantListAction> = () => (dispatch, state, cli
       dispatch(actions.refreshTenantList({ tenantList, currentTenantId: 0 }));
     }
     refreshData(dispatch);
+  }).catch(() => {
```

Comment:
OK, this explains my question why only switching between tenants detects a connection failure. I think every operation needs to do this.

Reply:
Issue is, operations can fail without meaning no connections. I think it better to handle generic connection failures in the RestServiceClient so we don't need to repeat the code everywhere